include/seqan3/io/alignment_file/header.hpp

```diff
@@ -140,23 +140,12 @@ class alignment_file_header
      *
      * | TAG | Description and Rules                                            |
      * | --- | ---------------------------------------------------------------- |
-     * | AH  | Indicates that this sequence is an alternate locus. The value is
-     *         the locus in the primary assembly for which this sequence is an
-     *         alternative, in the format 'chr:start-end', 'chr' (if known),
-     *         or '*' (if unknown), where 'chr' is a sequence in the primary assembly.
-     *         Must not be present on sequences in the primary assembly. |
-     * | AN  | Alternative reference sequence names. A comma-separated list
-     *         of alternative names that tools may use when referring to this
-     *         reference sequence. These alternative names are not used elsewhere
-     *         within the SAM file; in  particular, they must not appear in alignment
-     *         records’ RNAME or RNEXT fields. regular expression : name (, name )*
-     *         where name is [0-9A-Za-z][0-9A-Za-z*+.@ |-]* |
+     * | AH  | Indicates that this sequence is an alternate locus. The value is the locus in the primary assembly for which this sequence is an alternative, in the format 'chr:start-end', 'chr' (if known), or '*' (if unknown), where 'chr' is a sequence in the primary assembly. Must not be present on sequences in the primary assembly. |
```

Comment:
Is there a reason for these changes? They don't fit our style-guide of at most 120 chars per line.

Reply:
Resolves: "read_groups in alignment file in IO, barcode table is not correctly shown , same problem in ref_id_info"

Reply:
list 

* AH sdafsaaffa
* AN sdasdsa